<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Flipbook</class>
 <widget class="QDialog" name="Flipbook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>755</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Flipbook</string>
  </property>
  <layout class="QVBoxLayout" stretch="1,0">
   <item>
    <widget class="FlipbookView" name="view" native="true"/>
   </item>
   <item>
    <layout class="QHBoxLayout" stretch="0,0,1,0,0,0,0,0,0,0,1,1,1,0,0">
     <item>
      <widget class="QToolButton" name="refreshButton">
       <property name="toolTip">
        <string>Refresh</string>
       </property>
       <property name="icon">
        <iconset theme="view-refresh">
         <normaloff>theme:view-refresh.svg</normaloff>theme:view-refresh.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="zoomButton">
       <property name="toolTip">
        <string>Zoom</string>
       </property>
       <property name="icon">
        <iconset theme="zoom-out">
         <normaloff>theme:zoom-out.svg</normaloff>theme:zoom-out.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="popupMode">
        <enum>QToolButton::InstantPopup</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="rewindButton">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Rewind</string>
       </property>
       <property name="icon">
        <iconset theme="media-skip-backward">
         <normaloff>theme:media-skip-backward.svg</normaloff>theme:media-skip-backward.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="playButton">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Play</string>
       </property>
       <property name="icon">
        <iconset theme="media-playback-start">
         <normaloff>theme:media-playback-start.svg</normaloff>theme:media-playback-start.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="layerIndex">
       <property name="wrapping">
        <bool>true</bool>
       </property>
       <property name="correctionMode">
        <enum>QAbstractSpinBox::CorrectToNearestValue</enum>
       </property>
       <property name="showGroupSeparator" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel">
       <property name="text">
        <string>Range</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="loopStart">
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel">
       <property name="text">
        <string notr="true">-</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="loopEnd">
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="KisDoubleSliderSpinBox" name="speedSpinner">
       <property name="prefix">
        <string>Speed: </string>
       </property>
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
       <property name="maximum">
        <double>1000.000000000000000</double>
       </property>
       <property name="value">
        <double>100.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="exportButton">
       <property name="toolTip">
        <string>Export</string>
       </property>
       <property name="icon">
        <iconset theme="document-save-all">
         <normaloff>theme:document-save-all.svg</normaloff>theme:document-save-all.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Close</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FlipbookView</class>
   <extends>QWidget</extends>
   <header>desktop/widgets/flipbookview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>KisDoubleSliderSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>desktop/widgets/kis_slider_spin_box.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
